<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAiLearnClass</class>
 <widget class="QDialog" name="DlgAiLearnClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgAiLearn</string>
  </property>
  <widget class="QWidget" name="widget_create" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>是否为新纽扣?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_yes">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>是</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_no">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>否</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>点是将会删除之前</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存的纽扣图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_close">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>230</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_check" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pbtn_checkColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>检测颜色</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_checkKnifeShape">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>检测刀型</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_step" native="true">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>301</width>
     <height>821</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_stepTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>第一步：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stepDescribe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>放要筛选的纽扣</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_filterColorDiff">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>筛选色差</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_station1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>一工位</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_station2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二工位</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_station3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>三工位</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_station4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>四工位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_next">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>670</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_lookAllImage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查看纽扣图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_pre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_train">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>训练</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_test">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_pic" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_pic3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_progress" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>741</width>
     <height>171</height>
    </rect>
   </property>
   <widget class="QProgressBar" name="progress_learn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>训练中</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="txt_log">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>671</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_tranCompelete">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>135</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定完成</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
